index.html

```diff
@@ -14,55 +14,129 @@
             <div id="button-holder">
                 <img src="images/button.png" alt="Hamburger button" id="button">
             </div>
+            <div id="menubar">
+                <ul id="menu-list">
+                    <li><a class="anchor" href="#">Portfolio</a></li>
+                    <li><a class="anchor" href="#">About</a></li>
+                    <li><a class="anchor" href="#">Contact</a></li>
+                    <li><a class="anchor" href="#"><img src="images/mail.png" alt="Mail image"></a></li>
+                </ul>
+            </div>
         </nav>
     </header>
     <main id="content">
         <section id="container-1">
-            <h1 id="headline">Hey There.<br>I'm Amrit</h1>
-            <h2 id="title-line">I'am a Software Developer</h2>
-            <p id="para">I can help you build a product, feature or website Look through some of my work <br> and experience! If you like what you see <br> and have a project you need coded, don't hestiate to contact me.</p>
-            <ul class="socials-container">
-                <li class="socials">
-                    <a href="https://github.com/Amrit131" class="links"><img src="images/github.png" alt="github link" class="social-img"></a>
-                </li>
-                <li class="socials">
-                    <a href="https://www.linkedin.com/in/mishra-amrit/" class="links"><img src="images/linkedin.png" alt="linkedin link" class="social-img"></a>
-                </li>
-                <li class="socials">
-                    <a href="http://angellist.com" class="links"><img src="images/dnt.png" alt="dnt link" class="social-img"></a>
-                </li>
-                <li class="socials">
-                    <a href="https://twitter.com/i_ammrit" class="links"><img src="images/tweet.png" alt="Twitter link" class="social-img"></a>
-                </li>
-                <li class="socials">
-                    <a href="http://medium.com" class="links"><img src="images/microverse.png" alt="Microverse link" class="social-img"></a>
-                </li>
-            </ul>
-            <div id="bg-container" class="socials-container">
-                    <img id="bg-img" src="images/Groupbackground.png" alt="Background card">  
+            <div id="col-1">
+                <div id="first-content">
+                    <h1 id="headline">Hey There.<br>I'm Amrit</h1>
+                    <h2 id="title-line">I'am a Software Developer</h2>
+                    <p id="para">I can help you build a product, feature or website Look through some of my work <br> and experience! If you like what you see <br> and have a project you need coded, don't hestiate to contact me.</p>
+                </div>
+                <ul class="socials-container">
+                    <li class="socials">
```

Comment:
Good job, but as you know it's essential to follow the exact Figma template, so please make sure to add some space from left to your social icons.

![image](https://user-images.githubusercontent.com/34779928/194726845-70fcda1d-fd3d-4cdc-8e1b-33aa4385a2cb.png)

Reply:
# Implemented

![1st](https://user-images.githubusercontent.com/87594100/194728026-c27e2533-c770-47ab-9f1a-5e87f4c22f57.png)